bundles/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/actions/NotificationActionTypeProvider.java

```diff
@@ -34,41 +34,62 @@
  * openHAB Cloud.
  *
  * @author Christoph Weitkamp - Initial contribution
+ * @author Dan Cunningham - Extend2 additions
  */
 @NonNullByDefault
 @Component(service = ModuleTypeProvider.class)
 public class NotificationActionTypeProvider implements ModuleTypeProvider {
 
     private static final ModuleType SEND_NOTIFICATION_ACTION = new ActionType(SendNotificationActionHandler.TYPE_ID,
-            getSendNotificationConfig(false, null), "send a notification",
+            getNotificationConfig(ConfigType.NOT_EXTENDED, true, null), "send a notification",
             "Sends a notification to a specific cloud user.", null, Visibility.VISIBLE, null, null);
+
     private static final ModuleType SEND_EXTENDED_NOTIFICATION_ACTION = new ActionType(
-            SendNotificationActionHandler.EXTENDED_TYPE_ID, getSendNotificationConfig(true, null),
+            SendNotificationActionHandler.EXTENDED_TYPE_ID, getNotificationConfig(ConfigType.EXTENDED, true, null),
             "send a notification with icon and severity",
             "Sends a notification to a specific cloud user. Optionally add an icon or the severity.", null,
             Visibility.VISIBLE, null, null);
+
+    private static final ModuleType SEND_EXTENDED2_NOTIFICATION_ACTION = new ActionType(
+            SendNotificationActionHandler.EXTENDED2_TYPE_ID, getNotificationConfig(ConfigType.EXTENDED2, true, null),
+            "send a notification with icon, severity, on click, media attachment, action button 1, action button 2, action button 3",
```

Comment:
Looks better in the UI:

```suggestion
            "send a notification with icon, severity, on click, media attachment, action buttons",
```

Reply:
agreed